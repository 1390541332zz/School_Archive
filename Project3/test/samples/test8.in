File: test/maps/blacksburg_christiansburg.asc
Greatest Inclination 1
Least Elevation 20
Least Elevation 22
Greatest Inclination 6
Greatest Inclination 4