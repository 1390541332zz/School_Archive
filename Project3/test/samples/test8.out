Loading: test/maps/blacksburg_christiansburg.asc
Number of rows: 15
Number of columns: 20
	         0    1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16   17   18   19
	    0  746  766  790  688  608  622  672  613  554  537  626  617  597  604  591  597  589  589  602  671
	    1  753  772  808  699  709  756  775  673  547  557  627  632  569  587  573  585  614  662  641  724
	    2  783  799  788  791  826  806  759  634  575  529  610  560  512  554  583  647  662  745  884  943
	    3  805  797  780  838  881  814  706  607  506  542  505  521  576  611  710  843  898  912  863  768
	    4  757  707  745  748  828  782  659  570  512  559  597  531  737  866  813  754  700  682  600  574
	    5  743  667  628  668  671  587  570  586  689  813  801  605  659  585  522  545  554  547  542  542
	    6  703  656  622  591  639  645  686  866  916  727  601  516  516  565  602  561  538  571  574  546
	    7  648  687  693  684  719  792  918  845  687  558  534  628  692  833  995  857  600  594  564  540
	    8  957 1044  891  876  898  822  687  580  553  565  626  833  918  977  938  729  617  590  555  595
	    9 1035 1074  931  848  668  579  588  653  743  859  932 1022  970  771  669  635  617  594  556  586
	   10  814  718  673  649  724  837  932  992 1015 1041 1093  859  734  677  641  604  580  555  559  578
	   11  796  826  850  901  945  909  931  941 1062 1096  996  879  750  689  648  647  623  586  582  600
	   12  772  735  723  722  786  867  964 1045 1006  926  917  997 1047  894  752  618  603  560  584  638
	   13  723  701  682  755  827  880  886  866  794  784  823  825  756  709  658  603  575  607  707  751
	   14  707  690  679  763  847  819  798  772  845  797  653  622  611  637  699  704  634  694  687  708
	
Command: Greatest Inclination
	Height change:  190
	Steps:          26
	Average change: 7.30769
	Starting row:   1
Number of rows: 15
Number of columns: 20
	     0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19
	 0   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 1   \  -  \  -  -  -  v  \  -  >  >  >  >  \  -  -  -  -  -  -
	 2   -  /  -  \  -  /  v  ^  /  -  -  -  -  -  \  -  -  -  >  X
	 3   -  -  -  -  v  ^  v  ^  -  -  -  -  -  -  -  >  >  /  -  -
	 4   -  -  -  -  /  -  /  -  -  -  -  -  -  -  -  -  -  -  -  -
	 5   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 6   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 7   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 8   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 9   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	10   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	11   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	12   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	13   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	14   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	
Command: Least Elevation
Bad starting row: 20
Command: Least Elevation
Bad starting row: 22
Command: Greatest Inclination
	Height change:  240
	Steps:          29
	Average change: 8.27586
	Starting row:   6
Number of rows: 15
Number of columns: 20
	     0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19
	 0   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 1   -  -  \  -  -  -  v  \  -  >  >  >  >  \  -  -  -  -  -  -
	 2   -  /  -  \  -  /  v  ^  /  -  -  -  -  -  \  -  -  -  >  X
	 3   /  -  -  -  v  ^  v  ^  -  -  -  -  -  -  -  >  >  /  -  -
	 4   ^  -  -  -  /  -  /  -  -  -  -  -  -  -  -  -  -  -  -  -
	 5   ^  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 6   ^  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 7   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 8   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 9   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	10   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	11   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	12   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	13   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	14   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	
Command: Greatest Inclination
	Height change:  186
	Steps:          27
	Average change: 6.88889
	Starting row:   4
Number of rows: 15
Number of columns: 20
	     0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19
	 0   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 1   -  -  \  -  -  -  v  \  -  >  >  >  >  \  -  -  -  -  -  -
	 2   -  /  -  \  -  /  v  ^  /  -  -  -  -  -  \  -  -  -  >  X
	 3   /  -  -  -  v  ^  v  ^  -  -  -  -  -  -  -  >  >  /  -  -
	 4   ^  -  -  -  /  -  /  -  -  -  -  -  -  -  -  -  -  -  -  -
	 5   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 6   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 7   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 8   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 9   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	10   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	11   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	12   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	13   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	14   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	
